clusterloader2/api/types.go

```diff
@@ -106,6 +106,22 @@ type ListUnknownObjectOptions struct {
 	LabelSelector *metav1.LabelSelector `json: labelSelector`
 }
 
+// NamespaceConfig defines namespace parameters.
+type NamespaceConfig struct {
+	// Count is a number of automanaged namespaces.
+	Count int32 `json: count`
+	// NamePrefix is the name prefix of automanaged namespaces.
+	// It's optional, if set CL will use it, otherwise generate one with random string.
+	Prefix string `json: prefix`
+	// DeleteStaleNamespaces specifies whether or not delete stale namepaces
+	DeleteStaleNamespaces bool `json: deleteStaleNamespaces`
+	// DeleteAutomanangedNamespaces specifies whether or not delete namepaces after a test
+	DeleteAutomanagedNamespaces bool `json: deleteAutomanagedNamespaces`
+	// EnableExistingNamespace specifies whether to use existing namespaces
+	// if they exist  (TO DO)
+	EnableExistingNamespaces bool `json: enableExistingNamespaces`
```

Comment:
Let's not add fields that are not yet supported.

Reply:
will delete it.

Reply:
deleted it.